packages/api/cms-api/src/dam/files/files.controller.ts

```diff
@@ -42,8 +47,16 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
     const Scope = PassedScope ?? EmptyDamScope;
     const hasNonEmptyScope = PassedScope != null;
 
+<<<<<<< HEAD
     function nonEmptyScopeOrNothing(scope: DamScopeInterface): DamScopeInterface | undefined {
         return hasNonEmptyScope ? scope : undefined;
+=======
+    @Post("upload")
+    @UseInterceptors(DamUploadFileInterceptor(FilesService.UPLOAD_FIELD), FileValidationInterceptor)
+    async upload(@UploadedFile() file: FileUploadInterface, @Body() { folderId }: UploadFileBody): Promise<File> {
+        const uploadedFile = await this.filesService.upload(file, folderId);
+        return Object.assign(uploadedFile, { fileUrl: await this.filesService.createFileUrl(uploadedFile) });
+>>>>>>> main
```

Comment:
Remaining merge artifacts

Reply:
Strange, I was sure I already fixed this

Reply:
Ah it's because the branch is force pushed each time a new merge request is merged into main. So my changes were lost